### ISSUE TITLE
createWallet is blocking the UI thread on Android

### ISSUE THREAD

**Issue by testerez** [LtbLightning/bdk-rn#5]
I noticed that `BdkRn.createWallet(...)` is blocking the UI thread on Android.
Is it something you are aware about?

**Comment by BitcoinZavior** [comment]
Yes it is blocking for the first call, calls after the first one are faster. Is that what you have observed as well?
There is an overall effort to reduce the size of the binaries and also some efficiencies in building.
Will wait for that to complete and try again.

**Comment by BitcoinZavior** [comment]
This ticket will also help https://github.com/LtbLightning/bdk-rn/issues/12
